DFC.Digital/DFC.Digital.Web.Sitefinity/web.config.template

```diff
@@ -75,6 +75,9 @@
         <!-- Micro Services Publishing Endpoints-->
         <add key="DFC.Digital.MicroService.HelpEndPoint" value="__MicroServiceHelpEndPoint__" />
         <add key="DFC.Digital.MicroService.HelpPreviewEndPoint" value="__MicroServiceHelpPreviewEndPoint__" />
+      <!-- Servicebus Connection String-->
+      <add key="DFC.Digital.ServiceBus.ConnectionString" value="__ServiceBusCMSConnectionString__" />
+      <add key="DFC.Digital.ServiceBus.TopicName" value="__ServiceBusCMSTopicName__" />
```

Comment:
Make sure these are included in the parameters.xml

Reply:
Added